runtime/doc/lua.txt

```diff
@@ -1776,6 +1743,44 @@ vim.schedule_wrap({fn})                                  *vim.schedule_wrap()*
       • |vim.schedule()|
       • |vim.in_fast_event()|
 
+                                                         *vim.str_byteindex()*
+vim.str_byteindex({s}, {encoding}, {index}, {no_out_of_range})
```

Comment:
did you generate this with `make doc`? I wonder why it's out of order now. (not a blocker)